spec/test_doubles.rb

```diff
@@ -122,7 +122,7 @@ class NamedArgumentOrganiser
     extend LightService::Organizer
 
     def self.call(number:)
-      with(number: number).reduce([AddsTwoAction])
```

Comment:
Shouldn't we keep this to prove the functionality? Can you make a Rubocop exception for the next line here?

Reply:
I fear I'm not getting the point here, sorry :)

Let's firstly say that obviously wouldn't be a problem to drop a `rubocop:disable*`. Just wanna be sure to share comprehension.

But I think that the answer to the first question should be: no. To prove the functionality the ruby interpreter >=3 has to be able to parse the script and more specifically [this line](https://github.com/adomokos/light-service/pull/224/files#diff-17a3795f56e12613500ff7d30051980f41fa9f2017a7b466298c45c412b254edR30) w/o dropping an `ArgumentError`.

The error is dropped ATM because in >=3 you can't receive keyword arguments in a method with a signature such as `method(*args)`.

To prove the fix suffice to pass a keyword argument. AFAIK a keyword argument is a keyword argument disregarding the syntax used to write it.

BTW

- the test fails with ArgumentError with both syntax w/o the `ruby2_keywords` fix
- the test succeds w/o ArgumentError with both syntax with the `ruby2_keywords` fix

Reply:
Sorry all, I though I could wing this PR lazily through GH UI only...

Interestingly `:arg => value` is actually picked up as a kw argument:
![image](https://user-images.githubusercontent.com/867435/172851502-838ec5ca-d65a-4a70-bbdb-ccf274699d22.png)

However, that doesn't seem as evident as using directly the form `arg: value`. For clarity I'll use this form and add the rubocop rule.

Reply:
Just saying: that’s not that interesting to me, because

`:foo => ‘bar’`

Is the really same thing as

`foo: ‘bar’`

Having in both cases Symbol as key and String as value.

This is AFAIK and for the sake of discussing 🙂

Thank you all for your time and effort on this 🚀